build_rules.gradle

```diff
@@ -34,11 +34,19 @@ println "Applying build_rules.gradle to $project.name"
 /*************************************************************************************************/
```

Comment:
It seems like we changed the permissions of this file, can we reset it back to 100644?

Reply:
Opened https://github.com/swegner/beam/pull/19